homeassistant/components/bsblan/entity.py

```diff
@@ -0,0 +1,34 @@
+"""Base entity for the BSBLAN integration."""
+from __future__ import annotations
+
+from bsblan import BSBLAN, Device, Info
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.device_registry import format_mac
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import DOMAIN
+
+
+class BSBLANEntity(Entity):
```

Comment:
Why doesn't the BSBLANEntity entity inherit from CoordinatorEntity? There are no entities that don't need the coordinator yet.